server/i18n/en.json

```diff
@@ -5986,6 +5986,10 @@
     "id": "app.plugin.disabled.app_error",
     "translation": "Plugins have been disabled. Please check your logs for details."
   },
+  {
+    "id": "app.plugin.seek.app_error",
+    "translation": "Unable to reset the read position to the start of the plugin bundle."
+  },
```

Comment:
The order of the translations is not correct. You need to run `make i18n-extract` and commit the changes.